test/JDBC/expected/test_db_collation-vu-verify.out

```diff
@@ -0,0 +1,1800 @@
+~~END~~
+
+
+USE test_db_collation_vu_prepare_db141;
+GO
+
+SELECT name, collation_name from sys.columns WHERE name IN ('c', 'nc', 'v', 'nv', 't', 'nt', 'sqlv', 'sn') ORDER BY name + COALESCE(collation_name, '');
+GO
+~~START~~
+varchar#!#varchar
+c#!#bbf_unicode_cp1_ci_ai
+c#!#bbf_unicode_cp1_ci_ai
+c#!#bbf_unicode_cp1_ci_as
+c#!#bbf_unicode_cp1_ci_as
+nc#!#bbf_unicode_cp1_ci_ai
+nc#!#bbf_unicode_cp1_ci_ai
+nc#!#bbf_unicode_cp1_ci_as
+nc#!#bbf_unicode_cp1_ci_as
+nt#!#bbf_unicode_cp1_ci_ai
+nt#!#bbf_unicode_cp1_ci_ai
+nt#!#bbf_unicode_cp1_ci_as
+nt#!#bbf_unicode_cp1_ci_as
+nv#!#bbf_unicode_cp1_ci_ai
+nv#!#bbf_unicode_cp1_ci_ai
+nv#!#bbf_unicode_cp1_ci_as
+nv#!#bbf_unicode_cp1_ci_as
+sn#!#bbf_unicode_cp1_ci_ai
+sn#!#bbf_unicode_cp1_ci_ai
+sn#!#bbf_unicode_cp1_ci_as
+sn#!#bbf_unicode_cp1_ci_as
+sqlv#!#bbf_unicode_cp1_ci_ai
+sqlv#!#bbf_unicode_cp1_ci_ai
+sqlv#!#bbf_unicode_cp1_ci_as
+sqlv#!#bbf_unicode_cp1_ci_as
+t#!#bbf_unicode_cp1_ci_ai
+t#!#bbf_unicode_cp1_ci_ai
+t#!#bbf_unicode_cp1_ci_as
+t#!#bbf_unicode_cp1_ci_as
+v#!#bbf_unicode_cp1_ci_ai
+v#!#bbf_unicode_cp1_ci_ai
+v#!#bbf_unicode_cp1_ci_as
+v#!#bbf_unicode_cp1_ci_as
+~~END~~
+
+
+select column_name, collation_name FROM information_schema_tsql.COLUMNS WHERE TABLE_NAME = 'test_db_collation_vu_prepare_db141_t1' AND
+COLUMN_NAME IN ('c', 'nc', 'v', 'nv', 't', 'nt', 'sqlv', 'sn') ORDER BY COLLATION_NAME + COALESCE(collation_name, '');
+GO
+~~START~~
+nvarchar#!#nvarchar
+c#!#bbf_unicode_cp1_ci_as
+nc#!#bbf_unicode_cp1_ci_as
+v#!#bbf_unicode_cp1_ci_as
+nv#!#bbf_unicode_cp1_ci_as
+t#!#bbf_unicode_cp1_ci_as
+nt#!#bbf_unicode_cp1_ci_as
+sqlv#!#bbf_unicode_cp1_ci_as
+sn#!#bbf_unicode_cp1_ci_as
+~~END~~
+
+
+select column_name, collation_name FROM information_schema_tsql.COLUMNS WHERE TABLE_NAME = 'test_db_collation_vu_prepare_db141_t2' AND
+COLUMN_NAME IN ('c', 'nc', 'v', 'nv', 't', 'nt', 'sqlv', 'sn') ORDER BY COLLATION_NAME + COALESCE(collation_name, '');
+GO
+~~START~~
+nvarchar#!#nvarchar
+c#!#bbf_unicode_cp1_ci_ai
+nc#!#bbf_unicode_cp1_ci_ai
+v#!#bbf_unicode_cp1_ci_ai
+nv#!#bbf_unicode_cp1_ci_ai
+t#!#bbf_unicode_cp1_ci_ai
+nt#!#bbf_unicode_cp1_ci_ai
+sqlv#!#bbf_unicode_cp1_ci_ai
+sn#!#bbf_unicode_cp1_ci_ai
+~~END~~
+
+
+SELECT name, collation_name FROM SYS.TYPES WHERE collation_name IS NOT NULL ORDER BY name + COALESCE(collation_name, '');
+GO
+~~START~~
+varchar#!#varchar
+char#!#bbf_unicode_cp1_ci_as
+nchar#!#bbf_unicode_cp1_ci_as
+ntext#!#bbf_unicode_cp1_ci_as
+nvarchar#!#bbf_unicode_cp1_ci_as
+sql_variant#!#bbf_unicode_cp1_ci_as
+sysname#!#bbf_unicode_cp1_ci_as
+text#!#bbf_unicode_cp1_ci_as
+varchar#!#bbf_unicode_cp1_ci_as
+~~END~~
+
+
+CREATE TYPE ct FROM NVARCHAR(11);
+GO
+
+SELECT name, collation_name FROM SYS.TYPES WHERE name = 'ct';
+GO
+~~START~~
+varchar#!#varchar
+ct#!#bbf_unicode_cp1_ci_as
+~~END~~
+
+
+USE test_db_collation_vu_prepare_db142;
+GO
+~~ERROR (Code: 911)~~
+
+~~ERROR (Message: database "test_db_collation_vu_prepare_db142" does not exist)~~
```

Comment:
All the tests after this is unnecessary until next USE <db> statement

Reply:
Yes, this was added deliberately to test when server level collation is chines_prc_ci_as and user creates a database with that collation. May not be a very trivial use case, but added to confirm the behaviour. And that is the reason we have different sets of expected outputs.